app/types/scheduler.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from collections.abc import Callable, Coroutine
+from datetime import datetime, timedelta
+from typing import Any
+
+from arq import create_pool
+from arq.jobs import Job
+
+scheduler_logger = logging.getLogger("scheduler")
+
+
+async def create_scheduler(settings):
+    scheduler = Scheduler(settings)
+    if scheduler.settings.host != "":
+        await scheduler.async_init()
+    return scheduler
+
+
+class Scheduler:
+    """Disappears sometimes for no reason"""
+
+    def __init__(self, redis_settings):
+        self.settings = redis_settings
+
+    async def async_init(self):
+        if self.settings.host != "":
+            self.redis_pool = await create_pool(self.settings)
+            scheduler_logger.debug(f"Pool in init {self.redis_pool}")
+
+    async def queue_job_time_defer(
+        self,
+        job_function: Callable[..., Coroutine[Any, Any, Any]],
+        job_id: str,
+        defer_seconds: float,
```

Comment:
```suggestion
        defer_seconds: int,
```

Reply:
I think you could pass a timedelta directly. Other modules will probably construct a timedelta to deduce the number of seconds.

```suggestion
        defer_by: timedelta,
```